sql/core/src/main/scala/org/apache/spark/sql/execution/SortPrefixUtils.scala

```diff
@@ -46,4 +47,19 @@ object SortPrefixUtils {
       case _ => NoOpPrefixComparator
     }
   }
+
+  def getPrefixComparator(schema: StructType): PrefixComparator = {
+    val field = schema.head
+    getPrefixComparator(SortOrder(BoundReference(0, field.dataType, field.nullable), Ascending))
```

Comment:
Do we want to hardcode `Ascending` here? I realize that we only need to cluster, not sort, for aggregation, but in terms of API design maybe it would be clearer to fix `Ascending` in the KV sorter than here. If our KV sorter technically performs clustering instead of sorting then maybe we should add a comment to make that very explicit.